pandas/tests/scalar/timedelta/test_construction.py

```diff
@@ -195,28 +195,22 @@ def test_iso_constructor_raises(fmt):
         Timedelta(fmt)
 
 
-def test_td_constructor_on_nanoseconds():
+@pytest.mark.parametrize('constructed_td, conversion',
```

Comment:
Just stylistically if you put the left bracket to start the list of parameters on the first line it could make this more readable (reduces the indentation required for every subsequent line)

Reply:
very good point. thanks!